.github/workflows/tools-buildtest.yml

```
@@ -9,6 +9,7 @@ on:
   pull_request:
     branches:
       - master
+      - '[0-9][0-9][0-9][0-9].[0-9][0-9]-branch'
```

How about :

```suggestion
      - '*'
```

?

That would work for master and release branch, but also for feature branches that were already used in the past.

---

Good point!

---

Done.